acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/GenesisStateAcceptanceTest.java

```diff
@@ -39,4 +39,18 @@ public void shouldCreateTheSameGenesisState() throws Exception {
     // state because they processed the same deposits from the same ETH1 chain.
     lateJoinTeku.waitUntilInSyncWith(firstTeku);
   }
+
+  @Test
+  public void shouldCreateGenesisFromPartialDeposits() throws Exception {
+    final BesuNode eth1Node = createBesuNode();
+    eth1Node.start();
+
+    createTekuDepositSender().partiallySendValidatorDeposits(eth1Node, 4);
+
+    final TekuNode firstTeku = createTekuNode(config -> config.withDepositsFrom(eth1Node));
+    firstTeku.start();
+    firstTeku.waitForGenesis();
```

Comment:
Why would we get genesis if we only sent partial deposits? Shouldn't it send a second batch of deposits to reach the required amount?

Reply:
Oh I see, we bundled it all into one method.  It would make the test a lot easier to understand if there were actually two calls to the deposit sender here in the test.  We want the AT to read like a script showing each step that's being taken.

Reply:
Hopefully its more readable now.

Reply:
I noticed that with the changes after your comment, assemble fails because we don't have access to any object in the acceptance-test package other than all the object declared in the textFixtures package. That forces the acceptance tests to do almost all the work in a testFixtures object. Not sure how to make this acceptance test more readable in this scenario. Maybe I can make a TekuPartialDepositSender object, yet it feels a bit unnecessary.